src/scss/lr7-classic-theme/_toolbars.scss

```diff
@@ -266,6 +262,65 @@
 	}
 }
 
+// Management Toolbar Skin
+
+.management-toolbar-default {
+	background-color: $toolbar-default-bg;
+	border-bottom: 1px solid $toolbar-bottom-border;
+	padding: 0 24px;
+
+	.btn {
+		background-color: transparent;
+		border-width: 0;
+		padding: 0;
+	}
+
+	.dropdown-menu {
+		margin-top: $toolbar-dropdown-margin-top;
+	}
+
+	.toolbar-group, .toolbar-group-right, .toolbar-group-expand {
```

Comment:
A majority of this css could be omitted if we don't use class btn in the toolbar.